src/test/secretManagerService.test.ts

```diff
@@ -0,0 +1,242 @@
+import { SecretManagerService } from '../utils/SecretManagerService'
+import { logger as log } from '../utils/logger'
+import { configType } from '../models/Config'
+import NodeVault from 'node-vault'
+// Parsing configuration
+const config: configType = {
```

Comment:
i think we need to consolidate this config to a single place, import it into each test file and change we need. I don't think we need to copy/paste this into every test file like we're currently doing.

Reply:
done